lib/ecto/migration.ex

```diff
@@ -658,4 +659,17 @@ defmodule Ecto.Migration do
   defp validate_type!(%Reference{} = reference) do
     reference
   end
+
+  def prefix(opts_prefix) when is_nil(opts_prefix) do
```

Comment:
this fails because I'm setting the table prefix inside a quoted block in migration, that then gets sent to runner (where the private prefix functions aren't available).  I still need the prefix functions in migration for simply executed commands like just create(:table), but I think I need to duplicate them in runner.  I don't think this is a testing problem, I think its a real bug in the commit :-)